src/api/goods.ts

```diff
@@ -5,15 +5,26 @@ const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/good
 
 export function getAll(): Promise<Good[]> {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then(response => {
+      if (!response.ok) {
+        throw new Error('Loading failed');
+      }
+
+      return response.json();
+    });
 }
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods
+      .sort((good1, good2) => (
+        good1.name.localeCompare(good2.name)
+      ))
```

Comment:
bad practice to use a number in variable naming, I think we can use `a` and `b` in sort method.
It is normal so don't worry(but only for sort method)